napari_console/conftest.py

```diff
@@ -0,0 +1,67 @@
+import pytest
```

Comment:
to simplify this (particularly since it can all be deleted after napari/napari#2131), maybe we should _just_ make this single fixture, and put it in `test_qt_console` (and delete this conftest):

```python
@pytest.fixture
def make_test_viewer(qtbot, request):
    from napari import Viewer
     viewers = []

      def actual_factory(*model_args, viewer_class=Viewer, **model_kwargs):
         model_kwargs.setdefault('show', False)
         viewer = viewer_class(*model_args, **model_kwargs)
         viewers.append(viewer)
         return viewer

      yield actual_factory

      for viewer in viewers:
          viewer.close() 
```

Reply:
sounds good, will change now, and then will delete later

Reply:
i still needed a blank `conftest.py` for some reason ...